pipeline/lib/flood_model/dynamicDataDb.py

```diff
@@ -25,6 +25,8 @@ def __init__(self, leadTimeLabel, countryCodeISO3,admin_level):
         self.triggerFolder = PIPELINE_OUTPUT + "triggers_rp_per_station/"
         self.affectedFolder = PIPELINE_OUTPUT + "calculated_affected/"
         self.EXPOSURE_DATA_SOURCES = SETTINGS[countryCodeISO3]['EXPOSURE_DATA_SOURCES']
+        if countryCodeISO3 == 'MWI':
+            self.EXPOSURE_DATA_UBR_SOURCES = SETTINGS[countryCodeISO3]['EXPOSURE_DATA_UBR_SOURCES']
```

Comment:
what is EXPOSURE_DATA_SOURCES by the way can this be generic and maybe useful for other countries? if that is the case we can ass it for all countries and remove the if close

Reply:
I guess you meant EXPOSURE_DATA_UBR_SOURCES. If so, it is data about vulnerability of households collected and processed by Malawi govt. It is in tabular format. So far I am not aware if there is something similar in other countries. That's why I made it with an if clause so that this will not affect other countries.